### DISCUSSION TITLE
Update TVM to 1.19x

### DISCUSSION THREAD

**Discussion by MagmaSlime123** [eccentricdevotion/TARDIS#659] [General]
Title. Would be nice if there were more addons for the plugin, since Sonic Blaster is also terribly outdated, and could receive some changes to make it less OP.

**Comment by eccentricdevotion** [comment]
Updated TVM for Spigot 1.19.2 and TARDIS v4.11.0
http://tardisjenkins.duckdns.org:8080/job/TARDISVortexManipulator/

**Reply by MagmaSlime123** [reply]
Didn't know you still cared about TVM xD Thought it was fully dead
